packages/web/src/components/edit-playlist/mobile/EditPlaylistPage.tsx

```diff
@@ -238,6 +239,14 @@ const EditPlaylistPage = g(
 
         editPlaylist(metadata.playlist_id, editPlaylistData)
 
+        track({
+          eventName: Name.COLLECTION_EDIT,
+          properties: {
+            id: metadata.playlist_id,
+            to: formFields
```

Comment:
I can see how this data is useful, though I'm wondering if it makes sense to write every metadata update to our tracking service. It feels like that might be overkill or a lot of data over time. Tbh I'm unsure how the costs scale here, so will defer to those who do.